libs/dh/charges/feature-prices/src/lib/drawer/dh-charges-prices-drawer.component.html

```diff
@@ -21,17 +21,17 @@
       <span class="prices-label">{{ t("prices") }}</span>
       <watt-icon name="right"></watt-icon>
       <span
-        [ngClass]="{ clickable: (messageId$ | push) !== undefined }"
+        [ngClass]="{ clickable: (message$ | push) !== undefined }"
         (click)="goToCharge()"
         >{{ charge?.chargeId }}</span
       >
-      <ng-container *ngIf="(messageId$ | push) !== undefined">
+      <ng-container *ngIf="(message$ | push) !== undefined">
         <watt-icon name="right"></watt-icon>
-        <span>{{ messageId$ | push }}</span>
+        <span>{{ (message$ | push)?.messageId }}</span>
       </ng-container>
     </watt-drawer-topbar>
 
-    <watt-drawer-actions *ngIf="(messageId$ | push) === undefined">
+    <watt-drawer-actions *ngIf="(message$ | push) === undefined">
```

Comment:
Why does the visibility of _stop_ and _edit_ rely on `message` rather than `charge`?

Reply:
Because we don't want to show the drawer Actions when showing a Message.

I wanted to do this in a prettier way, but I couldn't figure out how.

Reply:
Maybe define a new boolean with a name that clearly tells whether the buttons should be visible, and then set the boolean in ts based on the `message`?

Reply:
Good idea, I will do that 👍

Reply:
I can see you where able to reuse an already existing variable 👍